_data/nav.yml

```diff
@@ -64,6 +64,9 @@
 
 - url: "configure"
   pages:
+  - url: "accessibility-options"
```

Comment:
This probably shouldn't be the first thing, as it's ordered in what people are most likely to use. Maybe move this down near localization?